src/student_success_tool/configs/schemas/pdp.py

```diff
@@ -1,139 +0,0 @@
-import typing as t
```

Comment:
dropped in favor of the "v2" schema, now located at `student_success_tool.schemas.pdp.project`